rebench.conf

```diff
@@ -261,6 +264,8 @@ experiments:
                 benchmark: ruby-indy
             - RBX314:
                 benchmark: ruby-rbx
+            - Topaz:
+                benchmark: ruby-rbx
```

Comment:
Is the `harness-rbx.rb` working better? Otherwise I'd recommend using `ruby-indy` or maybe `steady-ruby`.